### ISSUE TITLE
Some CLI errors with network parameters

### ISSUE THREAD

**Issue by wvh** [UpCloudLtd/upcloud-cli#279]
**Environment (required):**
- OS: [e.g. macOS (Darwin or M1), Linux, Windows]
- Version [e.g. 1.2.0, "upctl version" gives you this]

Version 3.2.1 on Arch Linux (amd64).

**Describe the bug**

The `network create` and `network modify` sub-commands return wonky error messages when trying to set the default router parameter.

```
upctl network create --name drima-de --zone de-fra1 --ip-network address=172.16.0.0/16,family=IPv4,dhcp=true,dhcp-default-route=true
✗ Creating network drima-de                                                                                                                                                                                                                                    
  Error: true is an invalid value for dhcp default rout, it can be true of false
```


**To Reproduce**
Try to create a network or modify its parameters.

**Expected behavior**
A clear and concise description of what you expected to happen (e.g. I wanted to create a server with these kinds of devices).

Set default route to true, so the Kubernetes create cluster command can be run.

**Observed behavior**
What happened instead (e.g. I received an error with a message of "BOOT_ORDER_INVALID").

Well, despite my best efforts to jumble the parameters around, that wonky error message kept on pestering me. It was very traumatising. Man, I don't even know why I'm at work so late on a Friday evening.

**Screenshots**

I don't think a text CLI error message needs screenshots.

**Additional context**

Worked around it by clicking things like an idiot in the Upcloud web UI.

**Comment by kangasta** [comment]
Thanks for the bug report! We will take a look at this 👀

**Comment by kangasta** [comment]
This should now be fixed in [v3.2.2](https://github.com/UpCloudLtd/upcloud-cli/releases/tag/v3.2.2)
